Core/Src/speed_calc.c

```diff
@@ -65,7 +65,7 @@ void speed_calc(void *argument) {
 		//caso a tarefa tenha sido chamada pela queue
 		case osOK:
 			//verifica se alguma roda está a muito tempo sem receber interrupção, caso sim a velocidade dessa roda é zerada
-			reset_speed_single(message, last_messages, min_count);
+			reset_speed_single(&message, &last_messages, min_count);
```

Comment:
`last_messages` já é um ponteiro
```suggestion
			reset_speed_single(&message, last_messages, min_count);
```